product_docs/docs/pgd/5/consistency/crdt/05_crdt-reset.mdx

```diff
@@ -7,7 +7,7 @@ Resetting CRDT values is possible but requires special handling. The asynchronou
 cluster means that different nodes might see the reset operation at different places in the change stream no matter how it's implemented. Different nodes might also initiate a reset concurrently, that is, before
 observing the reset from the other node.
 
-In other words, to make the reset operation behave correctly, it needs to be commutative with respect to the regular operations. Many naive ways to reset a value that might work well on a single-node fail
+In other words, for the reset operation to behave correctly, it needs to be commutative with respect to the regular operations. Many naive ways to reset a value that might work well on a single-node fail
```

Comment:
Wanted to verify the use of "naive" as it's not something I've seen before. Is this standard vocabulary? Or was "native" meant?

Reply:
@djw-m Would "basic" work?

Reply:
My suggestion would be "straightforward" or "obvious".